pandas_gbq/gbq.py

```diff
@@ -421,6 +439,17 @@ def sizeof_fmt(num, suffix='B'):
             num /= 1024.0
         return fmt % (num, 'Y', suffix)
 
+    def query_price_for(self, bytes_num):
+        figlet = None
+        price = bytes_num * self.query_price_for_TB / 2**40
```

Comment:
ok, so if there is no hard veto's (@jreback?!) i'll try to push this forward without figlet fonts